config/styleguide/config.js

```diff
@@ -7,6 +7,15 @@ const pkg = require('../../package.json');
 
 const componentsWithoutPathLine = ['Icons'];
 
+const redirectToDarkScript = `const redirectToDark = () => {
+  if (document.title.match(/DARK/) && !window.location.href.match(/theme=dark/)) {
```

Comment:
Haha funky, but ok.

Reply:
:-) Curious which part you thought was funky?  Having the code in a string? (Is there a better way?) Or the redirect logic?

Reply:
The code in a string, and checking the document title + lack of parameter in the URL to decide whether to redirect. I don't have better alternatives though, so my comment wasn't very useful.

Reply:
I like to hear comments like this even when you don't have suggestions to improve it.  As the code writer, it's not always obvious which code will be funky to others, so I think it's useful.

I was able to make an incremental improvement to the logic.  I took the title bit out, since it's now simpler to just add the script to the darkConfig file.